Runtime/Game/Requests/LootLockerLeaderboardDetails.cs

```diff
@@ -0,0 +1,152 @@
+        public LootLockerLeaderboardRewardPorgressionDetails details { get; set; }
+        public int amount { get; set; }
+        public string progression_id { get; set; }
+        public string reward_id { get; set; }
+
+    }
+
+    public class LootLockerLeaderboardRewardPorgressionDetails
+    {
+        public string key { get; set; }
+        public string name { get; set; }
+        public int amount { get; set; }
+        public string id { get; set; }
+    }
+
+    public class LootLockerLeaderboardRewardProgressionReset
+    {
+        public string created_at { get; set; }
+        public string updated_at { get; set; }
+        public LootLockerLeaderboardRewardProgressionResetDetails details { get; set; }
+        public string progression_id { get; set; }
+        public string reward_id { get; set; }
+    }
+    public class LootLockerLeaderboardRewardProgressionResetDetails
+    {
+        public string key { get; set; }
+        public string name { get; set; }
+        public string id { get; set; }
+    }
+
+    public class LootLockerLeaderboardRewardAsset
+    {
+        public string created_at { get; set; }
+        public string updated_at { get; set; }
+        public LootLockerLeaderboardRewardAssetDetails details { get; set; }
+        public string asset_variation_id { get; set; }
+        public string asset_rental_option_id { get; set; }
+        public int asset_id { get; set; }
+        public string reward_id { get; set; }
+        public string asset_ulid { get; set; }
+
+    }
+
+    public class LootLockerLeaderboardRewardAssetDetails
+    {
+        public string name { get; set; }
+        public string thumbnail { get; set; }
+        public string variation_name { get; set; }
+        public string rental_option_name { get; set; }
+        public int? variation_id { get; set; }
+        public int? rental_option_id { get; set; }
+        public int legacy_id { get; set; }
+        public string id { get; set; }
+    }
+
+    public class LootLockerLeaderboardReward
+    {
+        public string reward_kind { get; set; }
+        public LootLockerLeaderboardRewardPredicates predicates { get; set; }
+        public LootLockerLeaderboardRewardCurrency currency { get; set; }
+        public LootLockerLeaderboardRewardProgressionReset progression_reset { get; set; }
+        public LootLockerLeaderboardRewardProgression progression_points { get; set; }
+        public LootLockerLeaderboardRewardAsset asset { get; set; }
+
+    }
+
+    public class LootLockerLeaderboardRewardPredicates
+    {
+        public string id { get; set; }
+        public string type { get; set; }
+        public LootLockerLeaderboardRewardPredicatesArgs args { get; set; }
+
+    }
+
+    public class LootLockerLeaderboardRewardPredicatesArgs
+    {
+        public int max { get; set; }
+        public int min { get; set; }
+        public string method { get; set; }
+        public string direction { get; set; }
+
```

Comment:
A lot of your classes have an empty line at the end of them, why?

Reply:
It's a bad habit :P I use them for "prettiness" and also the thought of "I might add more in the future" but that doesn't really work for the SDK